libs/barista-components/style/themes.scss

```diff
@@ -49,6 +61,12 @@
   --dt-warning-default-color: #{$yellow-600};
   --dt-warning-hover-color: #{$yellow-700};
   --dt-warning-active-color: #{$yellow-800};
+  --dt-error-default-color: #{$red-600};
+  --dt-error-hover-color: #{$red-700};
+  --dt-error-active-color: #{$red-800};
+  --dt-recovered-default-color: #{$green-600};
+  --dt-recovered-hover-color: #{$green-700};
+  --dt-recovered-active-color: #{$green-800};
   --dt-cta-default-color: #{$green-600};
   --dt-cta-hover-color: #{$green-700};
   --dt-cta-active-color: #{$green-800};
```

Comment:
We are redefining these in every theme again and again. AS they should stay the same, maybe we want to move them into a mixin?

Reply:
Issue: #1056

Reply:
Thanks @rowa-audil for creating the issue #1056